cmd/maya-apiserver/app/server/volume_endpoint_v1alpha1.go

```diff
@@ -61,11 +61,15 @@ func (s *HTTPServer) volumeV1alpha1SpecificRequest(resp http.ResponseWriter, req
 
 	switch req.Method {
 	case "POST":
-		return volOp.create()
+		cvol, err := volOp.create()
+		volumeEvents(cvol, usage.VolumeProvision)
```

Comment:
Can we avoid `volume` in the method's name?
As we are already in a package which is specific to volume, we really do not need to have volume once more.

How about `sendEventOrIgnore` as the method name?